lib/src/main/kotlin/com/swmansion/starknet/data/types/transactions/Call.kt

```diff
@@ -0,0 +1,58 @@
+package com.swmansion.starknet.data.types.transactions
+
+import com.swmansion.starknet.data.selectorFromName
+import com.swmansion.starknet.data.types.Calldata
+import com.swmansion.starknet.data.types.Felt
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Call(
```

Comment:
Why was this moved? This is related to execution, it is not a transaction.

Reply:
It's a core part of `InvokeTransactionPayload` so I think it makes sense to keep them in the same module.

Reply:
But you also use call without **any transaction**, so having it here is misleading.

Reply:
You are correct about that. I guess the problem is it doesn't make much sense to bundle it with all other data classes in `execution` since it's used in many different places.

Perhaps we should just put `Call.kt` in top level `types` module to indicate that it is it's own separate thing?

Reply:
I started execution file as container for things related to execution and call is one of them. I see that over time we added things that are less relevant there, like GetBlockTransactionCountPayload. Maybe we should move the others to proper places and leave execution things as they are (Call, Calldata, ExecutionParams etc)?

Reply:
Perhaps having `Payloads.kt` file for all payloads defined in `Exectution.kt` would make sense. We could then keep call etc. in execution.

Though I don't know if it makes more sense than to extract call (and maybe calldata etc.) to separate file, keeping execution as it is?

Reply:
For me the problem is that payloads don't match execution category, so I would rather assign them to specific categories.